pdns/ws-recursor.cc

```diff
@@ -575,7 +576,7 @@ void RecursorWebServer::jsonstat(HttpRequest* req, HttpResponse *resp)
     resp->setBody(Json::object { { "entries", entries } });
     return;
   } else {
-    resp->setErrorResult("Command '"+command+"' not found", 404);
+    throw ApiException(ApiException::ErrGenericError, "Command '"+command+"' not found", 404);
```

Comment:
404 Not found is exclusive to HTTP world. Even though this is an HTTP interface, I don't believe error messages should be tied to do. IMO, it is better to have more generic error codes that do have a relation to the application itself rather than to HTTP protocol (moreover Not Found is already in http status message, so don't feel it is really needed here)